### DISCUSSION TITLE
Elm Electronics is no longer an operating company ?

### DISCUSSION THREAD

**Discussion by cr0i** [meatpiHQ/wican-fw#276] [General]
Hello @meatpiHQ ,

I´ve seen, that the Website https://www.elmelectronics.com/ gives the Info, that Elm Electronics dosn´t exist anymore?
What´s about elm327 protocol, is this a problem for WiCAN in the future?

**Comment by meatpiHQ** [comment]
@cr0i no thats not a problem, the business closed long time ago but now they shutdown the website. We are not reliant on elmelectonics in anyway. But It's sad to see them go..
